flink-table/flink-table-planner/src/test/java/org/apache/flink/table/sqlexec/SqlToOperationConverterTest.java

```diff
@@ -163,6 +169,229 @@ public void testSqlInsertWithStaticPartition() {
 		assertEquals(expectedStaticPartitions, sinkModifyOperation.getStaticPartitions());
 	}
 
+	@Test(expected = AssertionError.class) // TODO: tweak the tests when FLINK-13604 is fixed.
```

Comment:
Enable this test and comment the unsupported types?

Reply:
Agree, thanks ~